src/main/java/com/hotels/service/tracing/zipkintohaystack/ingresses/kafka/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 Expedia, Inc.
+ *
+ *       Licensed under the Apache License, Version 2.0 (the "License");
+ *       you may not use this file except in compliance with the License.
+ *       You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *       Unless required by applicable law or agreed to in writing, software
+ *       distributed under the License is distributed on an "AS IS" BASIS,
+ *       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *       See the License for the specific language governing permissions and
+ *       limitations under the License.
+ *
+ */
+package com.hotels.service.tracing.zipkintohaystack.ingresses.kafka;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.hotels.service.tracing.zipkintohaystack.forwarders.Fork;
+import com.hotels.service.tracing.zipkintohaystack.forwarders.haystack.SpanValidator;
+
+@ConditionalOnProperty(name = "pitchfork.ingress.kafka.enabled", havingValue = "true")
+@Configuration
+public class KafkaConsumerConfig {
+
+    @Bean
+    public KafkaConsumer<String, byte[]> kafkaConsumer(KafkaIngressConfig pitchForkConfig) {
```

Comment:
Can we change the name of the parameter from `pitchForkConfig` to `kafkaIngressConfig` or even `config` now that we've renamed this file?

Reply:
Yeap, makes total sense. Thanks!